### ISSUE TITLE
Any VIM like paste mode? 

### ISSUE THREAD

**Issue by sgon00** [helix-editor/helix#8028]
I am running helix in a ssh session.
Thus I can not use SPACE + p to paste clipboard. 
When I use system paste shortcut to paste clipboard in helix insert mode, the paste will be messed up with wrong auto-indentation. 
So is that possible to turn off the auto-indent temporarily or enable some kind of paste mode? 
Thanks a lot.

**Comment by pascalkuthe** [comment]
I don't think we want to add a paste mode. You need to install the ssh clipboard provider (or use a terminal with bracketed paste support)
